libsolidity/ast/ASTJsonExporter.cpp

```diff
@@ -332,31 +333,31 @@ bool ASTJsonExporter::visit(InheritanceSpecifier const& _node)
 {
 	setJsonNode(_node, "InheritanceSpecifier", {
 		std::make_pair("baseName", toJson(_node.name())),
-		std::make_pair("arguments", _node.arguments() ? toJson(*_node.arguments()) : Json::nullValue)
+		std::make_pair("arguments", _node.arguments() ? toJson(*_node.arguments()) : Json())
 	});
 	return false;
 }
 
 bool ASTJsonExporter::visit(UsingForDirective const& _node)
 {
-	std::vector<std::pair<std::string, Json::Value>> attributes = {
-		std::make_pair("typeName", _node.typeName() ? toJson(*_node.typeName()) : Json::nullValue)
+	std::vector<std::pair<std::string, Json>> attributes = {
+		std::make_pair("typeName", _node.typeName() ? toJson(*_node.typeName()) : Json())
 	};
 
 	if (_node.usesBraces())
 	{
-		Json::Value functionList;
+		Json functionList;
```

Comment:
Is there a specific reason for not creating this explicitlyas
```suggestion
		Json functionList = Json::array();
```
like you usually do in other places? Is the behaviour different if nothing is added below? Is it the same as before?

Reply:
True, maybe I should do this more consistently. However, it will automatically become an array here, once adding items.